pkgs/by-name/ff/ffpb/package.nix

```
@@ -0,0 +1,29 @@
+{
+  lib,
+  python3Packages,
+  fetchPypi,
+}:
+
+python3Packages.buildPythonPackage rec {
```

```suggestion
python3Packages.buildPythonApplication rec {
```
Since it is only used as a binary.

---

merged the suggestions + squashed into a single commit n_n